From a5bc0abb0d2b7b03f7642e7d90350de3a05944df Mon Sep 17 00:00:00 2001
From: Srikanth Srungarapu <ssrungarapu@cloudera.com>
Date: Fri, 4 Sep 2015 14:01:15 -0700
Subject: [PATCH 223/267] HBASE-14261 Enhance Chaos Monkey framework by adding
 zookeeper and datanode fault injections; ADDENDUM

Reason: Bug
Author: Srikanth Srungarapu
Ref: CDH-31173
---
 .../hadoop/hbase/zookeeper/ZKServerTool.java       |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hbase-server/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKServerTool.java b/hbase-server/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKServerTool.java
index f16c020..8a958c4 100644
--- a/hbase-server/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKServerTool.java
+++ b/hbase-server/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKServerTool.java
@@ -68,7 +68,8 @@ public class ZKServerTool {
    */
   public static void main(String args[]) {
     for(ServerName server: readZKNodes(HBaseConfiguration.create())) {
-      System.out.println("Zk host: " + server.getHostname());
+      // bin/zookeeper.sh relies on the "ZK host" string for grepping which is case sensitive.
+      System.out.println("ZK host: " + server.getHostname());
     }
   }
 }
-- 
1.7.9.5

